src/packages/block/block-grid/property-editors/block-grid-type-configuration/property-editor-ui-block-grid-type-configuration.element.ts

```diff
@@ -14,22 +16,60 @@ export class UmbPropertyEditorUIBlockGridTypeConfigurationElement
 	extends UmbLitElement
 	implements UmbPropertyEditorUiElement
 {
+	#context?: UmbPropertyDatasetContext;
+
 	@property({ attribute: false })
-	value: UmbBlockTypeBase[] = [];
+	value: UmbBlockTypeWithGroupKey[] = [];
```

Comment:
ideally in the block-grid folder, we make a types.js file that holds a UmbBlockGridType type. This interface would then extends UmbBlockTypeWithGroupKey.